tests/runtime/data/kubernetes/meta/options_use-kubelet-enabled.meta

```diff
@@ -0,0 +1,109 @@
+{
+	"apiVersion": "v1",
+	"kind": "Pod",
+	"metadata": {},
+	"items": [{
+		"metadata": {
+			"annotations": {
+				"prometheus.io/path": "/api/v1/metrics/prometheus",
+				"prometheus.io/port": "2020",
+				"prometheus.io/scrape": "true"
```

Comment:
not sure why github is showing this as red- may be because of tags instead of spaces or something?

Reply:
Seems like some github issue: https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view